### PR TITLE
Create getting_started_fr.md

### PR DESCRIPTION
**Pull request by Hsterofemia**
French Translation of getting_started_en, ready for review

### CODE REVIEW

**Comment by lh1008** [comment]
Pinging @Tomahna81 :)

File: monero-outreach-docs/translations/fr/archive/getting_started_fr.md [hunk 1]
```diff
@@ -0,0 +1,197 @@
+# Un Starting Kit pour Aider Monero
+
+**“Une communauté bouillonnante, éclatée aux quatre coins de la planète, composée d’individus et de groupes unis par un projet décentralisé: Monero” - 21 août, 2019**
```
Comment (Tomahna81) [inline monero-outreach-docs/translations/fr/archive/getting_started_fr.md:3]: La communauté de Monero est une communauté bouillonnante, éclatée aux quatre coins de la planète, composée d’individus et de groupes unis par un projet décentralisé

Reason: I feel that putting 'Monero' at the end move way to far the main topic of this sentence.

**Review by Tomahna81 — COMMENTED** [review]
Very good translation overall. Quite simple to review as well because the text is made of short sentences. I did not notice any typo or grammatical error. Well done !

**Comment by lh1008** [comment]
Pinging @Hsterofemia 

Hey Hsterofemia, 

There are already some reviews done by @Tomahna81. Could you please check them out? :)

File: monero-outreach-docs/translations/fr/archive/getting_started_fr.md (hunk 1 above)
Comment (Hsterofemia) [reply monero-outreach-docs/translations/fr/archive/getting_started_fr.md:3]: Done!

**Review by lh1008 — APPROVED** [review]
@Hsterofemia 

It was well done. Approved. Anyway the reviewer needs to approve the changes. I'll be pinging @Tomahna81 to review your changes too.

:)

File: monero-outreach-docs/translations/fr/archive/getting_started_fr.md (hunk 1 above)
Comment (Tomahna81) [reply monero-outreach-docs/translations/fr/archive/getting_started_fr.md]: All good.

**Review by Tomahna81 — APPROVED** [review]

**Pull request merged**